src/main/java/games/strategy/net/ServerMessenger.java

```diff
@@ -198,7 +198,7 @@ public String getPlayerMac(final String name) {
   // which can be very slow
   private final List<String> m_liveMutedUsernames = new ArrayList<>();
 
-  private boolean IsUsernameMuted(final String username) {
+  private boolean isUsernameMuted(final String username) {
```

Comment:
Yes!  I avoided changing the methods you already renamed in `IServerMessenger`.  I only touched non-overrides in `ServerMessenger` (at least that was my intention :smile:).